v4/src/md_scraper.test.js

```diff
@@ -0,0 +1,139 @@
+import { graphFromText } from './md_scraper'; // eslint-disable-line
```

Comment:
maybe move this to dedicated test dir?

Reply:
For what it's worth I've read that generally [unit tests are best placed next to the file they're testing](https://medium.com/@JeffLombardJr/organizing-tests-in-jest-17fc431ff850). It also helps keeps the import statements sane.

Reply:
Ah haven't done that before but that does make sense. We can move some of the files/directories around in the future if it gets too bloated

Reply:
Oh I didn't realize either -- sounds good though!